labcodes/django-react-webpack
# #55: Migrate to react hooks

**Link to task:** fix #55

**How to test:**
- run the app and make sure it's running as expected
- run jest tests
- run cypress tests

**Description of your solution:**
I migrated all class implementations to functions in react component files. To keep being able to test properly, I needed to remove enzyme (and all related configs) to @testing-library/react.
